src/Model.elm

```diff
@@ -72,14 +68,22 @@ type alias Draft =
     }
 
 
+type alias NotificationAggregate =
```

Comment:
It seems like a `Mastodon` internal type no? Why did you choose to put it in `Model` instead of `Mastodon`?